allennlp/data/dataset_readers/simple_language_modeling.py

```diff
@@ -52,8 +54,11 @@ def text_to_instance(self,  # type: ignore
                          sentence: str) -> Instance:
         # pylint: disable=arguments-differ
         tokenized = self._tokenizer.tokenize(sentence)
+        tokenized_with_ends = [Token(ELMoCharacterMapper.bos_token)]
+        tokenized_with_ends.extend(tokenized)
+        tokenized_with_ends.append(Token(ELMoCharacterMapper.eos_token))
```

Comment:
This part would be good to get a sanity check on. Any implications to just throwing these bos and eos tags back in?

Reply:
I should have said, "in" instead of "back in". In other words, do you foresee any complications to adding these directly instead of using the functionality provided by https://github.com/allenai/allennlp/blob/master/allennlp/data/token_indexers/token_characters_indexer.py#L29?